yarn.lock

```diff
@@ -4910,7 +4917,7 @@
 
 "@types/luxon@^2.3.2":
   version "2.3.2"
-  resolved "https://registry.yarnpkg.com/@types/luxon/-/luxon-2.3.2.tgz#8a3f2cdd4858ce698b56cd8597d9243b8e9d3c65"
+  resolved "https://registry.npmjs.org/@types/luxon/-/luxon-2.3.2.tgz"
```

Comment:
some of these locked dependencies were changed to use the npmjs registry instead of yarnpkg? o_0